main/streams/streams.c

```diff
@@ -1563,73 +1563,93 @@ PHPAPI zend_result _php_stream_copy_to_stream_ex(php_stream *src, php_stream *de
 			src->writepos == src->readpos) {
 		/* both php_stream instances are backed by a file descriptor, are not filtered and the
 		 * read buffer is empty: we can use copy_file_range() */
-		int src_fd, dest_fd, dest_open_flags = 0;
+		int src_fd, dest_fd, dest_open_flags;
```

Comment:
There might have been some warning in of the tool about potentially uninitialised memory but not 100% sure.

Reply:
Fair point. Should be cheap anyway, so I'll restore this.